resources/lang/en/user.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+return [
+
+    'admin' => 'Admin',
+    'handle_registrations' => 'Regisztrációk kezelése',
+    'accept' => 'Elfogad',
+    'reject' => 'Elutasít',
+    'show' => 'Mutat',
+    'status' => 'Status',
+    'personal_information' => 'Personal information',
+    'name' => 'Name',
+    'email' => 'E-mail',
+    'address' => 'Address',
+    'surname' => 'Surname',
+    'zip_code' => 'Zip code',
+    'city' => 'City',
+    'street' => 'Street',
+    'address_information' => 'Address information',
+    'language' => 'Language',
+    'date_and_place_of_birth' => 'Date and place of birth',
+    'mother_name' => "Mother's name",
+    'phone_number' => "Phone number",
+    'information_of_studies' => "Information of studies",
+    'name_of_secondary_school' => "Name of secondary school",
+    'year_of_the_leaving_exam' => "Year of the leaving exam",
+    'year_of_enrollment' => "Year of enrollment",
+    'neptun_identifier' => "Neptun identifier",
+    'faculty' => 'Faculty',
+    'jogtudomanyi_kar' => 'Jogtudományi Kar',
+    'barczi_gusztav_gyogypedagogiai_kar' => 'Bárczi Gusztáv Gyógypedagógiai Kar',
+    'bolcseszettudomanyi_kar' => 'Bölcsészettudományi Kar',
+    'informatikai_kar' => 'Informatikai Kar',
+    'pedagogiai_es_pszichologiai_kar' => 'Pedagógiai és Pszichológiai Kar',
+    'tanito_eso_vokepzo_kar' => 'Tanító- és Óvóképző Kar',
+    'tarsadalomtudomanyi_kar' => 'Társadalomtudományi Kar',
+    'termeszettudomanyi_kar' => 'Természettudományi Kar',
+    'workshop' => 'Workshop',
+    'angol_amerikai_muhely' => 'Angol-amerikai műhely',
+    'biologia_kemia_muhely' => 'Biológia-kémia műhely',
+    'bollok_janos_klasszika_filologia_muhely' => 'Bollók János Klasszika-filológia műhely',
+    'filozofia_muhely' => 'Filozófia műhely',
+    'aurelien_sauvageot_francia_muhely' => 'Aurélien Sauvageot Francia műhely',
+    'germanisztika_muhely' => 'Germanisztika műhely',
+    'informatika_muhely' => 'Informatika műhely',
+    'magyar_muhely' => 'Magyar műhely',
+    'matematika_fizika_muhely' => 'Matematika-fizika műhely',
+    'mendol_tibor_foldrajz-, föld- és környezettudományi műhely' => 'Mendöl Tibor földrajz-, föld- és környezettudományi műhely',
+    'olasz_muhely' => 'Olasz műhely',
+    'orientalisztika_muhely' => 'Orientalisztika műhely',
+    'skandinavisztika_muhely' => 'Skandinavisztika műhely',
+    'spanyol_muhely' => 'Spanyol műhely',
+    'szlavisztika_muhely' => 'Szlavisztika műhely',
+    'tarsadalomtudomanyi_muhely' => 'Társadalomtudományi műhely',
+    'törtenesz_muhely' => 'Történész műhely',
```

Comment:
These will be good for now as it is. We will make translations later.

@luksan47 Please create an issue for checking all translations.
(every supported language has the same number of keys in order with proper translations.)

Reply:
Will check when the main features are done, it's part of the release process.